Gemfile.lock

```diff
@@ -6,7 +6,7 @@ GIT
 
 GIT
   remote: git://github.com/chef/omnibus.git
-  revision: 3e59997f785ac9a7df177a7638265f6765d1d4b1
+  revision: 0286e3d8e5e4ec12cd4cb184832a66315de05b79
```

Comment:
Let's perform one more `bundle update omnibus` so we are using the just released `v5.0.0`.